cmd/webapp/webapp.go

```diff
@@ -35,15 +38,60 @@ func NewWebappCmd() *cobra.Command {
 }
 
 func serve(cmd *cobra.Command, args []string) {
-	engine := webapp.NewEngine()
-
-	s := &http.Server{
-		Addr:           fmt.Sprintf("%s:%d", host, port),
-		Handler:        engine,
-		ReadTimeout:    10 * time.Second,
-		WriteTimeout:   10 * time.Second,
-		MaxHeaderBytes: 1 << 20,
+	r := chi.NewRouter()
+
+	r.Get("/", IndexHandler)
+	r.Get("/home", func(w http.ResponseWriter, r *http.Request) {
+		w.Write([]byte("hi home"))
+	})
+
+	// Create a route along /files that will serve contents from
+	// the ./data/ folder.
+	workDir, _ := os.Getwd()
+	filesDir := http.Dir(filepath.Join(workDir, "webapp/frontend/assets/"))
+	FileServer(r, "/static", filesDir)
+
+	listenAddress := fmt.Sprintf("%s:%d", host, port)
+	err := http.ListenAndServe(listenAddress, r)
+	if err != nil {
+		log.Println("Error while serving HTTP:", err)
+	}
+	log.Printf("serving on port %s", listenAddress)
+
+}
+
+// Index data is used for the home template
+type Index struct {
+	Title string
+}
+
+func IndexHandler(w http.ResponseWriter, r *http.Request) {
+	data := Index{
+		Title: "Sapconsole",
```

Comment:
should be `SUSE Console for SAP Applications`

Reply:
adressed